styles/Juanda.css

```
@@ -0,0 +1,3 @@
+.section-juandavid{
+    color:rgb(104, 0, 0)
+}
```

Deberíamos agregar un espacio entre el nombre del selector y la llave y además el salto de línea al final del archivo.

```suggestion
.section-juandavid {
    color:rgb(104, 0, 0)
}

```